test-common/uk/gov/hmrc/economiccrimelevyaccount/generators/Generators.scala

```diff
@@ -0,0 +1,136 @@
+/*
```

Comment:
You can remove this file from the repo and also remove `Regex` as we don't need these for this backend